docs/mocksupport.rst

```diff
@@ -564,23 +564,37 @@ mode.
 Building the mock repository
 ----------------------------
 
+The mock repository should contain the CIM qualifier declarations, CIM classes,
+CIMInstances, and CIM methods to be used in the mock environment.
+
+:class:`~pywbem_mock.FakedWBEMConnection` and
+:class:`~pywbem_mock.DMTFCIMSchema` provide the tools to build the mock
+repository.
+
+The repository can be build by either defining the mof for the items to be
+inserted into the repository (either by defining the pywbem CIM Objects or by
+defining the MOF and compiling the mof) or by compiling qualifiers and classes
+directly from a DMTF CIM schema. A particular test repository may be a
```

Comment:
The sentence "This repository can be build ..." is confusing to me in the way it describes the different possibilities. The building of the repository supports two ways: from CIM objects and from MOF (which can be a string or a file, in both cases with include statements). For the building from MOF, the MOF files created by downloading the CIM Schema fall under the same category as user-created MOF files or MOF strings, and specifically are processed using the same methods of FakedWBEMConnection.

Reply:
Karl, when I wrote this comment, I had not yet discovered your compile_dmtf_schema() method on FakedWBEMConnection, and afterwards I forgot to update my comment. I'll read your new version then.